doc/howtos/booting-and-installation.rst

```diff
@@ -131,13 +131,12 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
    for recovery cases.)
 #. Plug in a micro-USB cable to the J3 connector on the board (corner next to the FTDI chip)
 #. Flip the DIP switch towards jumper J16
-#. Open :command:`minicom` or other terminal program on your host computer to attach to the serial console
-#. Download the ``flashall`` folder from the Ostro OS download folder for edison (on https://download.ostroproject.org)
-#. Copy the flashall script (``flashall.sh``) from the flashall folder to the Ostro OS image folder
-#. Then in the image folder run:: 
-
-    $ sudo ./flashall.sh
-
+#. Download the ``ostro-image`` or ``ostro-image-dev`` image from the Ostro OS download folder for edison (on https://download.ostroproject.org/releases/ostro-os/milestone/)
+#. Extract the image using the command ``$ tar xf ostro-image-*-edison-*.toflash.tar.bz2`` archive
+#. Change directory to the toFlash folder
+   **NOTE:** When flashing the ``-dev`` image please update ``IMAGE_NAME=`` in
+   the flashall.sh script. By default it will look for ``ostro-image``
```

Comment:
Ah.  When Ulf and I looked at the flashall.sh script, we didn't see -i in the usage description that's displayed.  On closer examination the -i and --image arguments are shown in the example command line that's displayed, but not listed in the usage description text that follows (that should be updated in the script, ja?).